navbar-component.html

```diff
@@ -22,15 +24,16 @@
       </div>
 
       <!-- Collect the nav links, forms, and other content for toggling -->
-      <div class="collapse navbar-collapse" id="br-navbar-collapse">
+      <div ng-if="!$ctrl.service.brandOnly" class="collapse navbar-collapse"
+        id="br-navbar-collapse">
```

Comment:
`id` is being used in the current implementation: https://github.com/digitalbazaar/bedrock-angular-navbar/blob/master/navbar-component.html#L25 

Are we looking to eliminate that usage as well?